README.md

```diff
@@ -48,3 +48,33 @@ Familiarize yourself with some system modelling concepts and cadCAD terminology.
 
 ## 4. Connect
 Find other cadCAD users at our [Discourse](https://community.cadcad.org/). We are a small but rapidly growing community.
+
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/BlockScience/cadCAD/graphs/contributors"><img src="https://opencollective.com/cadCAD1/contributors.svg?width=890&button=false" /></a>
```

Comment:
https://opencollective.com/cadCAD1/contributors.svg?width=890&button=false fails to load

Reply:
@monkeywithcupcake, we can merge the above if you can provide live source.